UI/src/stories/data/mappings.js

```diff
@@ -162,3 +163,7 @@ export const favoriteLocations = () => {
     return favA < favB ? -1 : favA > favB ? 1 : 0
   })
 }
+
+export const agencyQuestions = () => {
+  return questions
```

Comment:
We should map these since we don't want to use the prop names from the JSON.

`
    updateAgencyQuestions(state, value) {
      localStorage.removeItem('ripa_agency_questions')
      const agencyQuestions = value.agencyQuestions || null

      if (agencyQuestions) {
        const questions = value.agencyQuestions.map(item => {
          return {
            maxLength: item.MaxLength,
            label: item.Prompt,
            hint: item.Hint || null,
            required: item.Required,
            questionType: item.Type,
          }
        })
        if (questions.length > 0) {
          localStorage.setItem(
            'ripa_agency_questions',
            JSON.stringify(questions),
          )
        }
        state.agencyQuestions = questions
      }
    },
`